README.md

```diff
@@ -1,3 +1,147 @@
+
+# Feature summary
+
+**OpsGenie to Mattermost notifications:** Link your Mattermost channels with the OpsGenie Teams you want to see, so you and your team can get notifications about the creation and updates of each alert.
+
+# Set up
+
+## Installation HTTP
+
+To install, as a Mattermost system admin user, run the command ``/apps install http OPSGENIE_API`` in any channel. The ``/genie`` command should be available after the configuration has been successfully installed.
+
+The ``OPSGENIE_API`` should be replaced with the URL where the OpsGenie API instance is running. Example: ``/apps install http https://myapp.com/manifest.json``
+
+## Installation Mattermost Cloud
+
+To install, as a Mattermost system admin user, run the command ``/apps install listed genie`` in any channel. The ``/genie`` command should be available after the configuration has been successfully installed.
+
+
+## Configuration
+
+After [installing](#installation)) the app:
+1. Open your OpsGenie profile to get your credentials and link to your Mattermost instance.
+2. Select the **Integrations** tab in the **Settings** menu. Then, click on the **Add Integration** button.
+3. Inside the **Add Integration** menu, select the **API** option.
+4. Update the integration name and access. Finish the **API** integration set up clicking the **Save Integration** button.
+5. Copy the given API Key.
+6. Return to Mattermost. 
+7. As a Mattermost system admin user, run the ``/genie configure`` command.
+8. In the configuration modal, enter your API Key.
+
+# Admin guide
+
+## Slash commands
+
+- ``/genie configure``: This command will enable all the other commands; it asks the administrator for an API key (which will be used to execute calls to OpsGenie’s API).
+
+# End user guide
+
+## Get started
+
+## Use ``/genie`` commands
+
+- ``/genie help``: This command will show all current commands available for this application.
+- ``/genie alert create``: Allow any user to create a new alert.
+- ``/genie alert note``: Adds a note to an existing alert.
+- ``/genie alert close``: Closes an existing alert.
+- ``/genie alert ack``: Acknowledge an existing alert.
+- ``/genie alert unack``: UnAcknowledge an existing alert.
+- ``/genie alert snooze``: Snooze an existing alert for a period of time.
+- ``/genie alert assign``: Assign an existing to a mattermost team member.
+- ``/genie alert own``: Take ownership of an existing alert (assign alert to yourself).
+- ``/genie alert priority``: Set the priority of an existing alert.
+- ``/genie alert list``: Get a list of the existing alerts.
+- ``/genie team list``: Get a list of the existing teams.
+- ``/genie subscription add``: Creates a new subscription for notifications: choose a team and a channel and get notified of the updates in that team. You can subscribe more than one team per channel.
```

Comment:
We should make sure to differentiate between Mattermost teams and OpsGenie teams

```suggestion
- ``/genie subscription add``: Creates a new subscription for notifications: choose a OpsGenie team and a Mattermost channel and get notified of the updates in that team. You can subscribe more than one team per channel.
```

Reply:
We should use the current channel instead of having the user supply a channel as an argument